src/terminal.rs

```diff
@@ -420,7 +420,7 @@ where
         let height = height.min(self.last_known_size.height);
         self.backend.append_lines(height)?;
         let missing_lines =
-            height.saturating_sub(self.last_known_size.bottom() - self.viewport_area.top());
+            height.saturating_sub(self.last_known_size.height - self.viewport_area.top());
```

Comment:
`last_known_size` was a `Rect` that was only set from `with_options()` and `resize()`. Both did only have non-zero x,y for `Viewport::Fixed`. This method only runs for `Viewport::Inline(_)` and it is the only method that uses the field.